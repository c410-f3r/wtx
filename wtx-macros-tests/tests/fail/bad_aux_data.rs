#[wtx_macros::pkg(data_format(json), id(FooId))]
mod pkg {
  #[pkg::aux]
  impl Foo {
    #[pkg::aux_data]
    fn fdsfqw() {}
  }

  #[pkg::req_data]
  struct FooReq(
    i32
  );

  #[pkg::res_data]
  struct FooRes;
}

fn main() {
}
