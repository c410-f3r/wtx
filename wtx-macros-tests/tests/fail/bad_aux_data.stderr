error: This method must be named `foo_data`
 --> tests/fail/bad_aux_data.rs:6:8
  |
6 |     fn fdsfqw() {}
  |        ^^^^^^

error: aborting due to 1 previous error

